File: src/main/java/com/github/fbdo/pricing/FileUtils.java
package com.github.fbdo.pricing;

import java.io.File;

/**
 * Created by fabio on 28/09/14.
 */
public final class FileUtils {

    private FileUtils() {}

    public static File getScriptFolder(String exampleName) {
        File folder = new File("src/main/resources").getAbsoluteFile();
        File exampleFolder = new File(folder,
                exampleName);
        if (!exampleFolder.exists()) {
            throw new RuntimeException("The target folder does not exist, please create folder " + exampleFolder + " first");
        }

        return exampleFolder;
    }
}

File: src/main/java/com/github/fbdo/pricing/FileUtils.scala
package com.github.fbdo.pricing;

import java.io.File


/**
 * Created by fabio on 28/09/14.
 */
object FileUtils {

    def getScriptFolder(exampleName:String): File = {
        val folder = new File("src/main/resources").getAbsoluteFile
        val exampleFolder = new File(folder, exampleName);
        if (!exampleFolder.exists) {
            throw new RuntimeException("The target folder does not exist, please create folder " + exampleFolder + " first");
        }

        return exampleFolder;
    }
}
